assets/cluster-monitoring-operator/prometheus-rule.yaml

```diff
@@ -52,11 +54,11 @@ spec:
                 * on(namespace,workload,workload_type) group_left()
                 (
                   count without(pod) (namespace_workload_pod:kube_pod_owner:relabel{namespace=~"(openshift-.*|kube-.*|default|logging)"}) > 1
```

Comment:
`logging` could be removed, since default logging project is `openshift-logging`

Reply:
Make sense, but I will do that in a separate PR since this value comes from the namespace selector used for all the mixin. https://github.com/openshift/cluster-monitoring-operator/blob/master/jsonnet/main.jsonnet#L51

Reply:
I opened https://github.com/openshift/cluster-monitoring-operator/pull/1266